### ISSUE TITLE
On Start (before seeing a block) Signature Verification Error 

### ISSUE THREAD

**Issue by andrewnguyen22** [pokt-network/pocket-core#1196]
**Describe the bug**
On Start (before the node sees a block) many transactions are rejected with:
```
I[2020-11-30|14:19:23.337] Rejected bad transaction                     module=mempool tx=064DC9101C5361816B8CCE318CEAC2038F33E3BB2ACC4C17B6E6BC417D0F540D peerID=e09ce22e0abfd8129776128c0c9b3836024d8c6e res="&{CheckTx:code:4 log:\"{\\\"codespace\\\":\\\"sdk\\\",\\\"code\\\":4,\\\"message\\\":\\\"ERROR:\\\\nCodespace: sdk\\\\nCode: 4\\\\nMessage: \\\\\\\"signature verification failed for the transaction\\\\\\\"\\\\n\\\"}\" }" err=null
```
No consensus failure, but could be linked to https://github.com/pokt-network/pocket-core/issues/1081

This behavior seems similar to these described in the FAQ: https://github.com/pokt-network/pocket-core/issues/1172

@BenVanGithub would love some more details or theories on why these properties in the FAQ exist

```
Pocket has not processed a new block since being started. IE: pocket start
Pocket has not processed a new block since a validator change: IE: pocket accounts set-validator
in summary.
```
NOTE: this bug/error doesn't seem to affect consensus
